Marlin/Marlin_main.cpp

```diff
@@ -4829,6 +4844,66 @@ inline void gcode_M31() {
 
 #endif // SDSUPPORT
 
+#if ENABLED(PISTON_EXTRUDER)
+  inline void gcode_M35() { //turn on dryer fan
```

Comment:
It may be better to overload `M7`/`M8`/`M9` for the "dryer fan." We might implement `M36` and `M37` in the future.